apps/developer-hub/src/chapters/ChapterHelper/DAOChapter.tsx

```diff
@@ -0,0 +1,286 @@
+        ExchangeRateCoinsToSellPerCoinToBuy: 0.1,
+        QuantityToFill: 0.1,
+        OperationType:
+          DAOCoinLimitOrderOperationTypeString.DAOCoinLimitOrderOperationTypeStringASK,
+      } as Partial<DAOCoinLimitOrderWithExchangeRateAndQuantityRequest>;
+    },
+    method: deso.dao.CreateDAOCoinLimitOrder,
+    documentation: [
+      'https://docs.deso.org/for-developers/backend/transactions/construct-transactions/dao-transactions-api#create-dao-coin-limit-order',
+    ],
+    githubSource: [],
+    component: function () {
+      return (
+        <Route
+          key={this.title}
+          path={this.route}
+          element={
+            <Page
+              method={{
+                methodName: `deso.dao.CreateDAOCoinLimitOrder(request)`,
+                params: this.params,
+                method: this.method,
+              }}
+              pretext={PageSection(
+                this.title,
+                <div>Create a DAO Coin Limit Order</div>
+              )}
+              demo={true}
+              chapters={CHAPTERS}
+              selectedChapter={this}
+              bind="dao"
+            />
+          }
+        ></Route>
+      );
+    },
+  },
+  CANCEL_DAO_COIN_LIMIT_ORDER: {
+    parentRoute: ParentRoutes.dao,
+    title: 'Cancel DAO Coin Limit Order',
+    route: '/dao/cancel-dao-coin-limit-order',
+    params: () => {
+      return {
+        TransactorPublicKeyBase58Check: localStorage.getItem('login_key'),
+        CancelOrderID: "", // TODO: how do we want to show this working?
+      } as Partial<DAOCoinLimitOrderWithCancelOrderIDRequest>;
+    },
+    method: deso.dao.CancelDAOCoinLimitOrder,
+    documentation: [
+      'https://docs.deso.org/for-developers/backend/transactions/construct-transactions/dao-transactions-api#cancel-dao-coin-limit-order',
+    ],
+    githubSource: [],
+    component: function () {
+      return (
+        <Route
+          key={this.title}
+          path={this.route}
+          element={
+            <Page
+              method={{
+                methodName: `deso.dao.CancelDAOCoinLimitOrder(request)`,
+                params: this.params,
+                method: this.method,
+              }}
+              pretext={PageSection(
+                this.title,
+                <div>Cancel a DAO Coin Limit Order</div>
+              )}
+              demo={true}
+              chapters={CHAPTERS}
+              selectedChapter={this}
+              bind="dao"
+            />
+          }
+        ></Route>
+      );
+    },
+  },
+  GET_DAO_COIN_LIMIT_ORDERS:  {
+    parentRoute: ParentRoutes.dao,
+    title: 'Get DAO Coin Limit Orders',
+    route: '/dao/get-dao-coin-limit-orders',
+    params: () => {
+      return {
+        DAOCoin1CreatorPublicKeyBase58CheckOrUsername: localStorage.getItem('login_key'),
+        DAOCoin2CreatorPublicKeyBase58CheckOrUsername: "", // Empty string is for DESO.
```

Comment:
Let's remove this from the request object and apply the empty string in the library call IF they do not provide a value for DAOCoin2CreatorPublicKeyBase58CheckOrUsername.

Lets also do this for the other calls. I'll leave a comment for the changes in the lib file